packages/ipfs-gateway-race/lib/index.js

```diff
@@ -135,14 +137,15 @@ export async function gatewayFetch (
   gwUrl,
   cid,
   pathname,
+  search,
```

Comment:
Should we put the search as optional within options object?

Thinking that this change would mean a breaking change, as well as search params not being quite common. So, being forced to add a `''` string as param here seems weird.

Reply:
makes sense to me.  updated